exonum-java-binding-core/rust/src/proxy/service.rs

```diff
@@ -1,15 +1,16 @@
-use exonum::blockchain::{ApiContext, Service, Transaction};
+use exonum::api::{self, ServiceApiBuilder};
```

Comment:
```diff
-  use exonum::api::{self, ServiceApiBuilder}; 
+  use exonum::api::{self, ServiceApiBuilder, ServiceApiState};
```
?

Reply:
What's confusing you ? Yes, we begun to use `ServiceApiState` in closure at 159 line.